pkgs/racket-doc/pkg/scribblings/pkg.scrbl

```diff
@@ -840,8 +840,12 @@ the given @nonterm{pkg}s.
          #:changed "6.1.1.6" @elem{Added the @DFlag{no-trash} flag.}
          #:changed "6.4.0.14" @elem{Added the @DFlag{dry-run} flag.}
          #:changed "7.2.0.8" @elem{Added the @DFlag{recompile-only} flag.}
-         #:changed "7.4.0.4" @elem{Added the @DFlag{no-docs}, @Flag{D} flags.}]}
+         #:changed "7.4.0.4" @elem{Added the @DFlag{no-docs}, @Flag{D} flags.}
+         #:changed "8.14.0.2" @elem{Renamed from @command-ref{remove} to @command-ref{uninstall}.}]}
+
+@subcommand{@command/toc{remove}---A synonym for @command-ref{uninstall}.
```

Comment:
Nit: Although it may be typographically an abuse of em dashes, the convention overall here is to have spaces around `---`.

Reply:
Fixed! The range-diff is trivial but included:
```
1:  c259e01d05 = 1:  c259e01d05 prepare: make svn-style-command-line cases call procedures
2:  ebddf6625c = 2:  ebddf6625c prepare: support #:alias declarations in svn-style-command-line
3:  aa1a8c6fee = 3:  aa1a8c6fee prepare: push alias support into pkg/commands
4:  abfc30132b = 4:  abfc30132b pkg: dedent top-level forms
5:  54c51a4a54 ! 5:  81355921f0 raco pkg: rename remove to uninstall
    @@ pkgs/racket-doc/pkg/scribblings/pkg.scrbl: the given @nonterm{pkg}s.
     +         #:changed "7.4.0.4" @elem{Added the @DFlag{no-docs}, @Flag{D} flags.}
     +         #:changed "8.14.0.2" @elem{Renamed from @command-ref{remove} to @command-ref{uninstall}.}]}
     +
    -+@subcommand{@command/toc{remove}---A synonym for @command-ref{uninstall}.
    ++@subcommand{@command/toc{remove} --- A synonym for @command-ref{uninstall}.
      
     +@history[#:changed "8.14.0.2" @elem{Made @command-ref{remove} an alias.}]}
      
6:  46965135e1 = 6:  853f171b91 pkg uninstall: update user messages for new command name
```